File: app/src/main/java/com/example/allianceapp/ui/sportclub/SportclubViewModel.java
package com.example.allianceapp.ui.sportclub;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class SportclubViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public SportclubViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("Спортклуб");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/example/allianceapp/ui/sportclub/SportclubViewModel.kt
package com.example.allianceapp.ui.sportclub

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class SportclubViewModel : ViewModel() {
    private val mText: MutableLiveData<String>
    val text: LiveData<String>
        get() = mText

    init {
        mText = MutableLiveData()
        mText.value = "Спортклуб"
    }
}